packages/commonwealth/client/scripts/views/pages/user_dashboard/TrendingCommunitiesPreview/TrendingCommunitiesPreview.tsx

```diff
@@ -27,26 +38,23 @@ export const TrendingCommunitiesPreview = () => {
         !['"', '>', '<', "'", '/', '`'].includes(name[1])
       );
     })
-    .map((community) => {
-      const isMember = Permissions.isCommunityMember(community.id);
-
-      return {
-        community,
-        monthlyThreadCount: +community.recentThreadsCount,
-        isMember,
-        // TODO: should we remove the new label once user visits the community? -- ask from product
-        hasNewContent: (data?.joinedCommunityIdsWithNewContent || []).includes(
-          community.id,
-        ),
-        onClick: () => navigate(`/${community.id}`),
-      };
-    })
+    .map((community) => ({
+      community,
+      isMember: Permissions.isCommunityMember(community.id),
+      hasNewContent: (
+        newContent?.joinedCommunityIdsWithNewContent || []
+      ).includes(community.id || ''),
+      onClick: () => navigate(`/${community.id}`),
+    }))
     .sort((a, b) => {
```

Comment:
shouldnt we move this sorting as well to the backend, actually in this PR?

Reply:
actually, its a bit tricky to merge this atm, because of `hasNewContent`, but u r right, we'sd probably do this. It might involve deleting the GetNewContent query tho.